### DISCUSSION TITLE
Reports not working

### DISCUSSION THREAD

**Discussion by DasGregier** [netbox-community/netbox#17298] [Help Wanted!]
Good morning,
I am getting this error when i open  the reports

![NetBox Report Error](https://github.com/user-attachments/assets/cd9b9300-3ac0-46a8-b97e-7a8bc25c441b)

**Comment by candlerb** [comment]
You have a bad report. It doesn't tell you which one though; you might be able to see if you get a full backtrace (set `DEBUG=True` in `configuration.py` and restart)

Otherwise, if you think you know which one it is, you can replace it with a zero-byte file to get past this:

```
cd /opt/netbox/netbox/reports
mv suspect_report.py suspect_report.py.orig
touch suspect_report.py
```
